src/_gettsim/social_insurance_contributions/ges_pflegev.py

```diff
@@ -247,13 +372,45 @@ def ges_pflegev_beitr_selbst_m(
     return out
 
 
-def ges_pflegev_beitr_rente_m(
+@dates_active(
+    start="1995-01-01", end="2004-12-31", change_name="ges_pflegev_beitr_rente_m"
+)
+def ges_pflegev_beitr_rente_m_ohne_zusatz_fuer_kinderlose(
+    _ges_krankenv_bemessungsgrundlage_rente_m: float,
+    ges_pflegev_beitr_satz: float,
+) -> float:
+    """Calculating the contribution to health insurance for pension income.
+
+    Pensioners pay twice the contribution of employees.
+
+    Parameters
+    ----------
+    _ges_krankenv_bemessungsgrundlage_rente_m
+        See :func:`_ges_krankenv_bemessungsgrundlage_rente_m`.
+    ges_pflegev_beitr_satz
+        See :func:`ges_pflegev_beitr_satz`.
+
+    Returns
+    -------
+    Monthly health insurance contributions for pension income.
+
+    """
+    out = _ges_krankenv_bemessungsgrundlage_rente_m * (ges_pflegev_beitr_satz * 2)
```

Comment:
I read [here](https://www.spiegel.de/wirtschaft/urteil-rentner-muessen-pflegeversicherung-allein-bezahlen-a-451421.html#) that until March 2004, the public pension fund did pay half of the Pflegeversicherungsbeitrag. I think this is not implemented here.

Reply:
Excellent catch! Would prefer to do that separately, though. Can you open an issue and maybe a PR?